helix-term/src/ui/menu.rs

```diff
@@ -52,6 +55,45 @@ impl Item for PathBuf {
 
 pub type MenuCallback<T> = Box<dyn Fn(&mut Editor, Option<&T>, MenuEvent)>;
 
+pub struct FileChangeData {
```

Comment:
Let's move this into the command definition as well, similar to how buffer_picker and jumplist_picker define their Item types within the command. #9647 will eliminate the dependency of the Picker on menu::Item so this type doesn't necessarily belong here, and I think this type will be local to the changed file picker anyways. (For an in-document picker of changes we will probably not need the untracked style for example.)

Reply:
Fixed with `7fd75b50`.